system/jlib/jptree.cpp

```diff
@@ -7774,20 +7766,26 @@ static void applyCommandLineOption(IPropertyTree * config, const char * option,
     }
 }
 
-static void applyCommandLineOption(IPropertyTree * config, const char * option)
+static void applyCommandLineOption(IPropertyTree * config, const char * option, std::list<std::string> ignoreOptions)
```

Comment:
efficiency: It doesn't really matter in this context, but if the parameters are going to be constant then std::initializer_list is lower overhead.  This could be std::initializer_list<const char *>.

Reply:
ok, I may change it.